threema-connector/config/variables.yaml

```diff
@@ -0,0 +1,11 @@
+Variables:
+  connector:
+   
+    # Your Threema.Gateway ID 
+    threemaId: ''
+    
+    # Your Threema.Gateway Secret
+    secret: ''
```

Comment:
declare it as password by stating `# [password]` ; in order to mask this value on our configuration UI's, such as the engine-cockpit
https://dev.axonivy.com/doc/10.0/designer-guide/configuration/variables.html

Reply:
added password annotation